libnavui-androidauto/build.gradle

```diff
@@ -41,15 +41,15 @@ dependencies {
     // This defines the minimum version of Maps, Navigation, and Search which are included in
     // this SDK. To upgrade the SDK versions, you can specify a newer version in your downstream
     // build.gradle.
-    api("com.mapbox.navigation:android:2.5.0-rc.1")
+    api("com.mapbox.navigation:android:2.6.0-alpha.2")
     api("com.mapbox.search:mapbox-search-android:1.0.0-beta.29")
+    implementation project(":libnavui-app")
```

Comment:
Seems this may resolve to `2.7.0-SNAPSHOT`

Reply:
Ah since this is `implementation` the version does not entirely matter. If it were `api` then this would matter

Reply:
Are you sure we can do this? `:libnavui-app` won't be compiled into Android Auto AAR. I'm afraid this might cause runtime exceptions due to missing classes.

Reply:
This should definitely depend on a concrete version of the `libnavui-app`, since `libnavui-app` module will be released alongside the rest of the SDK, not alongside the Android Auto artifact.